data/human/ships.txt

```diff
@@ -155,8 +155,8 @@ ship "Arrow"
 			"hull damage" 120
 			"hit force" 360
 	outfits
-		"Sidewinder Missile Launcher"
-		"Sidewinder Missile" 45
+		"Sidewinder Missile Pod" 2
```

Comment:
Yes. It struck me that the arrow, being a VIP transport, might make sense to be able to do a bit more than a stock fighter (the arrow has nowhere to restock, whereas a fighter by definition has either a carrier or a home base to restock from)

And the Hai variant also has two pods.

Reply:
Makes sense about adding the extra pod. I just brought up the Hai variant because it already had two pods prior to your changes, so I wasn't sure if we would want to maintain its (slight) armament advantage over the stock version by bumping it to 3 or 4 pods. Definitely not a big deal either way.